app/controllers/bulk_updates_controller.rb

```diff
@@ -0,0 +1,46 @@
+# :nodoc:
+class BulkUpdatesController < ManageMetadataController
+
+  add_breadcrumb 'Bulk Updates', :bulk_updates_path
+
+  def index
+    @provider_id = current_user.provider_id
+
+    @tasks = []
+
+    # We cannot implement pagination yet, because the cmr dummy responses do not include it
+    # when the response has `hits` and accepts page information, we should implement it in accordance with other resources (i.e. groups)
+    filters = {}
+
+    # filters[:page_size] = RESULTS_PER_PAGE
+    # default page to 1
+    # page = params.fetch('page', 1)
+    # @filters[:page_num] = page.to_i
+
+    bulk_updates_list_response = cmr_client.get_bulk_updates_list(@provider_id, filters, token)
+
+    if bulk_updates_list_response.success?
+      @tasks = bulk_updates_list_response.body.fetch('tasks', [])
+    else
+      Rails.logger.error("Error retrieving Bulk Updates Jobs List: #{bulk_updates_list_response.inspect}")
+      flash[:error] = Array.wrap(bulk_updates_list_response.body['errors'])[0]
```

Comment:
No flash errors on pages that users have not taken an action on. These should only be used when the user has submitted a form or some other action where they COULD resolve the issue by modifying the data they are submitting